### ISSUE TITLE
NTR: Precipitous labor  (< 4 hours) (Endocrine & genitourinary)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7957]
New term request Precipitous labor  (< 4 hours) (elthat_101012001637):   UMLS: 2136473

**Comment by pnrobinson** [comment]
@MickeySegal
I looked at a few sources, and the definition of precipitous labor seems to have 3 hours (not 4) as the threshold.
https://pubmed.ncbi.nlm.nih.gov/30940371/

Adding new term: 
- ID: HP:6000103
- label: Precipitous labor
- parents: [Pregnancy history (HP:0002686)](https://hpo.jax.org/app/browse/term/HP:0002686)
- pmid: PMID:25584099|PMID:30940371
- def: Precipitous labor is extremely rapid labor and delivery, usually defined as expulsion of the fetus within less than 3 hours of commencement of regular contractions.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7957
- synonyms: Precipitous delivery(NONE)
